lib/metrics_exporter.js

```diff
@@ -28,6 +28,10 @@ function isUp(status) {
     return ((status === 'UP') ? '1' : '0');
 }
 
+function ms2s(ms) {
```

Comment:
I'd prefer a more obvious name like msToSec, or millisecondsToSeconds, tho I suspect the long form is not worth the formatting hassle.

Reply:
Renamed `ms2s()` to `msToSec()`